packages/nimble-components/src/button/specs/README.md

```diff
@@ -0,0 +1,45 @@
+# Nimble-Button New Visual Designs Implementation
+
+## Problem Statement
+
+The Visual Design spec for Nimble buttons now includes primary button states `primary-mono` and `primary` that aren't implemented in the nimble-button component yet.
+
+## Links To Relevant Work Items and Reference Material
+
+[Nimble Issue #663: Adopt nimble-button visual designs for states primary, quiet, and loud](https://github.com/ni/nimble/issues/663)
+
+[Figma Spec](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-77205)
+
+## Implementation / Design
+
+[FAST Button API](https://github.com/microsoft/fast/blob/de7f234ef871204fcac2b5df59433d919809341d/packages/web-components/fast-foundation/src/button/README.md)
+
+The appearance of the button will have states `default` (White), `primary-mono` (Black), and `primary` (Green), each with its own color scheme based on the UI theme.
+`primary-mono` and `primary` states of buttons are primary buttons, so there should only be one of either in a section.
+
+The `primary-mono` button can be used when there is a conflict with color and its context.
+The `primary` button can be used in situations where a button needs to have the most prominent eye-catching approach, or when there is a lack of color.
+
+Ghost buttons will not have `primary-mono` or `primary` states.
+Outline and Block buttons in the Color UI will not have a `primary` state, as this has been declared as not recommended for this type of background.
+
+The current appearance-variant attribute will be used to hold these new states, as its previous implementation purpose is no longer used.
```

Comment:
For clients who are currently using the `primary` appearance variant, the proposal is that they'll automatically migrate to the new `primary` variant. Could you double check that Brandon is comfortable with us forcing that transition on all apps? (He usually comes to our team meeting on Thursdays so we can ask him then or you can reach out offline). I doubt it'll be a problem since it's just a slight change in the shade of green, but want to be sure.

Reply:
Sounds good. I'll reach out to him offline to ask a few smaller questions and give him a head start on larger ones, and we can continue the discussion with everyone at today's team meeting.